### PR TITLE
June 6 Release Notes

### PR DESCRIPTION
**Pull request by lbadenhop**
Thank you for your contribution to the Amplify-Central repo.

## Describe the changes

Enter a brief description of your changes to communicate to the reviewers what you changed and why.

## Deploy preview link

Please add the deploy preview link to the **specific page** that you've changed.

## Checklist for contributors

Before submitting this PR:

* Verify that all status checks have passed
* For more information on the Markdown linter rules, see [DavidAnson/markdownlint](https://github.com/DavidAnson/markdownlint)
* For first-time contributors, ensure that you have signed the [Axway CLA](https://cla.axway.com/)

### CODE REVIEW

**Comment by docsbuilder** [comment]
### <span aria-hidden="true">✅</span> Deploy preview is ready!

 |  Name | Link |
 |---------------------------------|------------------------|
 |<span aria-hidden="true">🔨</span> Latest commit | dd198bf |
 |<span aria-hidden="true">😎</span> Deploy preview | http://PR-310.amplify-central.opendocs-builder.pcloud.axway.int |

File: content/en/docs/amplify_relnotes/20230606_amplify.md [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+
+The following enhancements and bug fixes are available in this release.
+
+#### Feature updates for Service Registry
+
+There are no new features in this release.
+
+#### Bug fixes for Service Registry
+
+* **Team Ownership list** - When a user has multiple roles on a team, they could not view all team ownership teams when publishing an API to the Marketplace via the Web UI. This is now fixed.
```
Comment (georgiana-caras) [inline content/en/docs/amplify_relnotes/20230606_amplify.md:81]: When a user with multiple roles was publishing to the Marketplace an API Service, they could not see all teams ownership. This is now fixed.

File: content/en/docs/amplify_relnotes/20230606_amplify.md [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+
+The following enhancements and bug fixes are available in this release.
+
+#### Feature updates for Asset Catalog
+
+There are no new features in this release.
+
+#### Bug fixes for Asset Catalog
+
+* **Draft Asset Activation** - When a user has multiple roles on a team, they could not activate a draft asset via the Web UI. This is now fixed.
```
Comment (georgiana-caras) [inline content/en/docs/amplify_relnotes/20230606_amplify.md:101]: **Draft Asset Activation** - A user with multiple roles in a team could not activate an asset in draft state via the Web UI. This is now fixed.

File: content/en/docs/amplify_relnotes/20230606_amplify.md [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+
+#### Feature updates for Product Foundry
+
+* **UI improvements** - The consumer org team is visible in the Subscriptions/Credentials screen.
+* **Billing Integration** - When defining a quota type, you can create a standard loose quota type that is different from the billing period. Monthly plan billing is not allowed with "Annual" loose quota types.
+
+#### Bug fixes for Product Foundry
+
+* **Image upload** - Adding an image resulted in a black background. This is now fixed to behave correctly.
+* **Products display** - A placeholder image has been added to the product card when none is set to avoid inconsistencies.
```
Comment (georgiana-caras) [inline content/en/docs/amplify_relnotes/20230606_amplify.md:119]: **Products display** - A placeholder image has been added to the product card when none is set to be consistent with other image placeholders in the Web UI.

**Pull request merged**